src/utils/domUtils.ts

```diff
@@ -107,6 +107,28 @@ export module _Util {
       if (boxA.top > boxB.bottom) { return false; }
       return true;
     }
+
+    export function containedInBoundingBox(container: ClientRect, element: ClientRect) {
+      return (
+        Math.floor(container.left) <= Math.ceil(element.left) &&
+        Math.floor(container.top)  <= Math.ceil(element.top)  &&
+        Math.floor(element.right)  <= Math.ceil(container.left + container.width) &&
+        Math.floor(element.bottom) <= Math.ceil(container.top  + container.height)
+      );
+    }
+
+    export function getBoundingSVG(elem: D3.Selection) {
```

Comment:
Two things:
1. Please update `isSelectionRemovedFromSVG()` to use this implementation.
2. There's a much faster way to do this:
```Typescript
var svg = elem.node().ownerSVGElement;
return (svg == null) ? elem.node() : svg;
```